software/base/src/main/java/org/apache/brooklyn/entity/software/base/lifecycle/WinRmExecuteHelper.java

```diff
@@ -82,16 +82,29 @@ public WinRmExecuteHelper setPsCommand(String psCommand) {
      * _and_ block until completion, throwing on error */
     @Beta
     public Task<Integer> queue() {
-        return DynamicTasks.queueIfPossible(newTask()).orSubmitAndBlock().getTask();
+        return queue(false);
+    }
+
+    private Task<Integer> queue(boolean suppressFailures) {
```

Comment:
Again, why is `suppressFailures` a parameter on `queue()`, rather than config you can set on the `WinRmExecuteHelper`? I'd favour adding a method `suppressFailures(boolean)`.